info.json

```diff
@@ -205,6 +205,19 @@
                 }
             },
             "retired": "n"
+        },
+        "marktennyson":{
+            "name": "Aniket Sarkar",
+            "bio": ["passionate python and javascript developer"],
+            "volunteer": {
+                "education": {},
+                "code": {}
```

Comment:
All good, just

- Remove code. Code is added after consequent contributions to Pallets-eco or helping out with [FlaskCon's website this year](https://www.reddit.com/r/flask/comments/ph5cex/flaskcon_is_looking_for_volunteers/).
- Open a new issue here: https://github.com/flaskcwg/join-requests to be added. When done, i'll merge it. Thanks A LOT